src/db/migrations/20250207185525_add_data_broker_id_column_to_qa_custom_brokers.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /**
+  * @param { import("knex").Knex } knex
+  * @returns { Promise<void> }
+  */
+ export function up(knex) {
+  return knex.schema
+    .table("qa_custom_brokers", table => {
+      table.string('data_broker_id').nullable();
```

Comment:
is this supposed to be unique field?
Is this necessary? Since I think `data_broker` field is already unique and identifying the row?